### ISSUE TITLE
I've lost 40+ frames please help!

### ISSUE THREAD

**Issue by muse79** [opentoonz/opentoonz#3725]
**Describe the bug**
I had created an animation and was duplicating frames and alterating them, upon a crash, after opening several frames in the level replaced with "no icon" sign. One frame even has a drawing, but still displays a "no icon" sign. How do I restore the frames, is it possible?

**To Reproduce**
List steps to reproduce the behavior.
Create level with frames
Duplicate some frames
Crash during duplication and on repoen unaltered frames were missing with "no icon" sign.

**Expected behavior**
Loss of some random frames with a "no icon" sign.

**Screenshots**
![image](https://user-images.githubusercontent.com/79422068/108638107-8376b780-7485-11eb-8788-ded50fa05dca.png)

**Device Information:**
 - OS: Windows 10 Home
 - OpenToonz Version: 1.4
 - GPU: AMD Radeon R5 M435
 - GPU Driver Version: 27.20.1034.6

**Comment by artisteacher** [comment]
If you are lucky, there may be backups in the drawings folder. Using your computer's file browser (ie. Finder or File Explorer), look in the drawings folder for the missing drawings with .bak at the end of the name. Rename them without .bak. If you re-open the scene or reload the level, they might be back.

I like to turn up the number of backups to 2 in preferences for cases like this.

**Comment by muse79** [comment]
I changed the file names but unfortunately they are exactly the same - missing frames :(. I did, however, render the animation - so I could use the render and piece it together in a video editor, so not all is lost.

I am very worried that this may happen again tho.
How do backups work - do they save at the point of autosave? Because it would be good to be able to have previous backups. I may save extra files every few days and do extra renders!
